gradle/performanceTest.gradle

```diff
@@ -196,18 +196,25 @@ tasks.withType(DistributedPerformanceTest) {
     teamCityUrl = "https://builds.gradle.org/"
     teamCityUsername = project.findProperty("teamCityUsername")
     teamCityPassword = project.findProperty("teamCityPassword")
+    afterEvaluate { p ->
+      if (branchName) {
+        channel = channel + "-" + branchName
```

Comment:
How do you know what `+` will do at compile time when you can change the behavior at runtime? :)